bin/pint

```diff
@@ -150,22 +151,93 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        # in case of no results, return empty JSON
+        # instead of empty dictionary
+        data_json = {'images': []}
+        data_xml = etree.Element('images')
+        for reg in region.split(','):
+            data = ifsrequest.get_image_data(
+                framework,
+                image_state,
+                output_format,
+                reg.strip(),
+                command_args['--filter']
+            )
+            if '<images/>' in data or '"images": []' in data:
+                continue
+
+            if 'json' in output_format:
+                data = json.loads(data)
+                if data.get('images')[0] not in data_json['images']:
+                    # add new elements only
+                    data_json['images'] += data.get('images')
+            elif output_format in ['xml', 'plain']:
+                data_content = data.partition("<images>")[-1][:-10].strip()
```

Comment:
Magic slice numbers deserve a comment, what is `-10` cutting off?

Reply:
Done